docs/src/_static/storytangl-type.css

```
@@ -0,0 +1,20 @@
+ * Three families, three roles, never mixed.
+ *
+ *   Newsreader       — prose, titles, editorial copy, the wordmark
+ *   JetBrains Mono   — engine output, CLI, inline code, fragment data
+ *   Inter Tight      — UI chrome, nav, small labels, metadata
+ *
+ * No fourth family. No display-script. No "fun" override for headings.
+ * If a context needs typographic emphasis, reach for weight or size,
+ * never an additional family.
+ */
+
+@import url('https://fonts.googleapis.com/css2?family=Newsreader:ital,opsz,wght@0,6..72,400;0,6..72,500;0,6..72,600;0,6..72,700;1,6..72,400&family=JetBrains+Mono:wght@400;500;600;700&family=Inter+Tight:wght@400;500;600;700&display=swap');
```

Stylelint (17.12.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Newsreader:ital,opsz,wght@0,6..72,400;0,6..72,500;0,6..72,600;0,6..72,700;1,6..72,400&family=JetBrains+Mono:wght@400;500;600;700&family=Inter+Tight:wght@400;500;600;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Newsreader:ital,opsz,wght@0,6..72,400;0,6..72,500;0,6..72,600;0,6..72,700;1,6..72,400&family=JetBrains+Mono:wght@400;500;600;700&family=Inter+Tight:wght@400;500;600;700&display=swap'" (import-notation)

(import-notation)